applications.go

```diff
@@ -0,0 +1,95 @@
+package solus
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
+type ApplicationsService service
+
+type LinkType string
+
+const (
+	None    LinkType = "none"
+	URL     LinkType = "url"
+	JS_CODE LinkType = "js_code"
```

Comment:
Should be `LoginLinkTypeJSCode`. Use this as an example https://github.com/solusio/solus-go-sdk/blob/master/icon.go#L7-L11